Headers/AppLovinSDK.framework/Headers/ALEventService.h

```diff
@@ -0,0 +1,89 @@
+//
+//  ALEventService.h
+//  AppLovinSDK
+//
+//  Created by Thomas So on 2/13/19
+//  Copyright © 2020 AppLovin Corporation. All rights reserved.
+//
+
+#import "ALEventTypes.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Service that tracks various analytical events.
+ */
+@interface ALEventService : NSObject
+
+/**
+ * Sets a super property that this service will record with all future events.
+ *
+ * If `superProperty` is `nil`, this method will remove the super property with key `key` from being recorded with all future events.
+ *
+ * @param superProperty The value to assign to the super property whose key is `key`. Valid types include `NSString`, `NSNumber`, `NSSDate`, `NSURL`, `NSArray`,
+ *                      and `NSDictionary`. Set this to `nil` to remove the super property whose key is `key` from being recorded with all future events.
+ * @param key           The key that identifies the the super property whose value this method sets.
+ */
+ // [PLP] what does it mean to "record" a property (where is it recorded, how can this record be accessed)?
```

Comment:
"record" as in, every future event that the pub tries to track with this class, we'll pass up this "super property" in the request to our backend, which eventually will allow for aggregation based on it.

Like if pub sets gender superproperites, they can aggregate event data based on gender... However, we ended up not building this feature in our UI, so no need to go into details.